Sources/App/Core/Github.swift

```diff
@@ -82,35 +82,24 @@ enum Github {
             let repoUri = try? apiUri(for: package, resource: .repo),
             let issuesUri = try? apiUri(for: package, resource: .issues, query: ["state": "closed",
                                                                                  "sort": "closed",
-                                                                                 "direction": "desc"])
+                                                                                 "direction": "desc",
+                                                                                 "page_size": "100"]),
```

Comment:
I've also opened up issue fetching to 100 per page. This was perhaps the reason we saw some gaps in the "last closed" fields. I.e. if all last 30 closed items were of only one kind, the last closed date of the other kind would be `nil`, as they are pushed off the first page.

This is something we could also correct by fetching the next page conditionally but I think the better approach is to revamp this to use the GraphQL API.